modules/tensor_mechanics/src/materials/crystal_plasticity/ComputeMultipleCrystalPlasticityStress.C

```diff
@@ -77,6 +82,8 @@ ComputeMultipleCrystalPlasticityStress::ComputeMultipleCrystalPlasticityStress(
     _line_search_method(getParam<MooseEnum>("line_search_method").getEnum<LineSearchMethod>()),
     _plastic_deformation_gradient(declareProperty<RankTwoTensor>("fp")),
     _plastic_deformation_gradient_old(getMaterialPropertyOld<RankTwoTensor>("fp")),
```

Comment:
Can we switch to full names for those material properties? 

`fp` --> `plastic_deformation_gradient`
`pk2` --> `second_piola_kirchhoff_stress`
...

Reply:
Sure, I can make this change.